dds_cli/project_status.py

```diff
@@ -148,6 +148,112 @@ def update_status(self, new_status, deadline=None, is_aborted=False, no_mail=Fal
 
         dds_cli.utils.console.print(f"Project {response_json.get('message')}")
 
+    def extend_deadline(self):
+        """
+        Extend the project deadline
+        """
+        # Define initial parameters
+        extra_params = {"send_email": False}
+
+        # Fetch project status and default deadline
+        response_json, _ = dds_cli.utils.perform_request(
+            endpoint=DDSEndpoint.UPDATE_PROJ_STATUS,
+            headers=self.token,
+            method="patch",
+            params={"project": self.project},
+            json=extra_params,
+        )
+
+        # Structure of the response:
+        #   {
+        #   'default_unit_days': 30,
+        #   'project_info': {
+        #       'Created by': 'First Unit User',
+        #       'Description': 'This is a test project',
+        #       'Last updated': 'Wed, 18 Oct 2023 08:40:43 GMT',
+        #       'PI': 'support@example.com',
+        #       'Project ID': 'project_1',
+        #       'Size': 0,
+        #       'Status': 'Available',
+        #       'Title': 'First Project'
+        #       },
+        #   'project_status': {
+        #       'current_deadline': 'Sat, 04 Nov 2023 23:59:59 GMT',
+        #       'current_status': 'Available'},
+        #   'warning': 'Operation must be confirmed before proceding.'
+        #   }
+
+        # Extract default unit days and current deadline
+        default_unit_days = response_json.get("default_unit_days")
+        current_deadline = response_json.get("project_status").get("current_deadline")
+
+        # print information about the project status and table with the project info
+        print_info = (
+            f"\nCurrent deadline: [b][green]{current_deadline}[/green][/b]\n"
+            f"Default deadline extension: [b][green]{default_unit_days}[/green][/b] days\n"
+        )
+        table = self.generate_project_table(project_info=response_json.get("project_info"))
+        dds_cli.utils.console.print(table)
+        dds_cli.utils.console.print(print_info)
+
+        # First question, number of days to extend the deadline
+        prompt_question = (
+            f"Enter the number of days you want to extend the project, "
+            f"the number of days has to be equal or same as "
+            f"[b][green]{default_unit_days}[/green][/b].\n"
+            f"Or leave it empty to apply the default "
+            f"[b][green]{default_unit_days} days [/green][/b]"
+        )
+
+        dds_cli.utils.console.print(prompt_question)
+        extend_deadline = rich.prompt.Prompt.ask("-")
+        if not extend_deadline:
+            # Set extend_deadline to default
+            extend_deadline = default_unit_days
```

Comment:
* Only ask if the user hasn't already specified a number
* Rephrase the question
* We can just use the `ask` to print out this time (and at the same time use the `default` option), don't need the `print` first.
```suggestion
        if not extend_deadline: 
             # First question, number of days to extend the deadline
             prompt_question = (
                 f"How many days would you like to extend the project deadline with? "
                 f"Leave empty in order to choose the default ([b][green]{default_unit_days}[/green][/b])."
             )

             extend_deadline = rich.prompt.Prompt.ask(prompt_question, default=default_unit_days)
```